typescript/README.md

```
@@ -46,7 +42,7 @@ Or from microphone (it requires [arecord](https://www.thegeekdiary.com/arecord-c
 npm run live-microphone <your_gladia_key>
 ```
 
-Note: You can get your gladia key from https://app.gladia.io.
+Note: You can get your Gladia key from https://app.gladia.io.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)